### ISSUE TITLE
Zombie Horde Area Tool Idea

### ISSUE THREAD

**Issue by Kagutrenchi** [Zet0rz/nzombies#425]
So, when you down, usually via being surrounded, (especially in solo) you are usually royally screwed. In solo, there zombies will continue surrounding you, you could try despawning them, yes, but there's a chance the zombies won't despawn. So, instead of having to suffer the consequences of perkless surrounding (or even zombie blood surrounding) why not have the zombies go to another area. This idea is used in CoD:Zombies because Treyarch and Infinity Ward are merciful. So, how this could work is you have a tool to mark a place, this place shows up in Creative mode as an icon (in case you need to find one to undo) and would be a low priority target, having players high priority. If a zombie can not find a player who is up without a zombie blood, it would go to a close by horde area until it can find a player.

**Comment by Zet0rz** [comment]
I want to do something like this, but I don't think it should be a tool. Instead it should just be automatic targeting of some navmesh a certain distance away.

**Comment by minin43** [comment]
Have them return to spawn?

**Comment by lolleko** [comment]
Or a hiding spot on a nearby navarea.

**Comment by minin43** [comment]
Except the script or config dev has to set that, returning to spawn is 100x easier and better than what we have currently

**Comment by lolleko** [comment]
> Except the script or config dev has to set that

No, hiding spots are generated with the navmesh. There should be more hiding spots than spawn points, this could lead to a better distrubution also they wouldnt run to far away. But idk probably doesnt matter too much.

**Comment by Kagutrenchi** [comment]
@lolleko Well, that's fine. It's best hard to recover from, but survivable. Don't want it too easy or impossible.

**Comment by minin43** [comment]
I highly doubt those "hiding spots" will be accurate when the play area may be drastically changed from the original map. It's just a bunch more work I don't want to do.

**Comment by lolleko** [comment]
Define accurate? I already said hiding spots exist on any map with a navmesh, so there is no extra work for a config creator.

**Comment by Zet0rz** [comment]
Hiding spots sounds like a good idea. We can probably require them to be a certain distance away as well. Could probably even combine spawnpoints too if no hiding spots where found.

> Except the script or config dev has to set that

Hiding spots are generated with the navmesh automatically.

> I highly doubt those "hiding spots" will be accurate when the play area may be drastically changed from the original map

I think he's talking about when you use props to block off areas as well as editing the navmesh. Anyway, despite that, as long as there is still _some_ navmesh area that was generated in the first place, it's gonna have hiding spots. It will also only navigate to those it can navigate to obviously. Also worth noting, hiding spots aren't actually hiding spots, they're more just corners and small areas of navmeshes.

**Comment by jaxjaxk** [comment]
@Zet0rz it really does not matter that hiding spots are generated or not. the only way the game runs well is if you modify the navmesh manually. so we already have to go in why not take the extra step XD
